src/js/iotjs.js

```diff
@@ -55,21 +54,19 @@ Native.prototype.compile = function() {
 global.console = Native.require('console');
 global.Buffer = Native.require('buffer');
 
-(function() {
```

Comment:
What was the need for this function removal? With this the _timeoutHandler can be now accessed from any js file, and also the timers variable.